src/InviteRenderer.js

```diff
@@ -157,13 +158,30 @@ module.exports = class InviteRenderer {
     const badgeContainer = innerContainer.nested().y(2)
 
     // Feature Badges
-    invite.guild.features.forEach(f => {
-      if (BADGES[f]) {
-        const flowerStar = badgeContainer.path(Constants.SPECIAL_BADGE).fill(themeColors.badges[f].flowerStar)
-        badgeContainer.path(BADGES[f]).fill(themeColors.badges[f].icon)
-        EXTRA_SERVER_NAME_PADDING = flowerStar.width() + BADGE_MARGIN_RIGHT
-      }
-    })
+    if (invite.guild.features.includes('VERIFIED') && invite.guild.features.includes('HUB')) {
```

Comment:
If the guild is unknown it throws an undefined error, that's kinda expected, but I think it can be fixed with verifying if the server exists before verifying if it includes the 'VERIFIED' feature

Reply:
I've added a check for whether or not the guild exists after fetching the invite and made it return 404 if so